packages/@sanity/form-builder/src/sanity/uploads/types.ts

```diff
@@ -32,4 +32,11 @@ export type Uploader = {
   priority: number
 }
 
-export type UploaderResolver = (type: SchemaType, file: File) => Uploader | null
+export interface FileIsh {
+  // mime type
+  type: string
+  // file name (e.g. somefile.jpg)
+  name?: string
+}
```

Comment:
I feel like the common pattern for these "duck-types" is to use the "Like" suffix, eg PromiseLike - could we name this FileLike instead?

Reply:
Yeah, good call!